ui/ducks/locale/locale.js

```diff
@@ -13,7 +14,25 @@ export default function reduceLocaleMessages(state = {}, { type, payload }) {
   }
 }
 
-export const getCurrentLocale = (state) => state.localeMessages.currentLocale;
+/**
+ * @param state
+ * @returns {string} one of the codes in file://./../../../app/_locales/index.json.
+ * These codes are not safe to use with the Intl API.
+ */
+export const getLocaleNotSafeForIntl = (state) =>
```

Comment:
For the sake of minimising refactor and the PR diff, is this really needed if `SimulationDetails` was the first usage of `Intl.NumberFormat` with one of these locales, and there was no issues previously?

Would the above warning in the JSDoc, plus the presence of the `getIntlLocale` method be a sufficient deterrent?

Reply:
Done.